packages/sdk/src/across/feeCalculator.ts

```diff
@@ -0,0 +1,86 @@
+// Taken from https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/helpers/acrossFeesCalculator.ts
```

Comment:
Going to assume this file doesn't need review.

Reply:
yep just copied directly